CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.1.0](https://github.com/paritytech/substrate-api-sidecar/compare/v1.0.0...v1.1.0) (2020-11-17)
+
+
+### ⚠ BREAKING CHANGES
+
+* Bump polkadot-js and document runtime/metadata API regression (#338)
```

Comment:
Ideally users would be able to learn what is breaking exactly; maybe link to the pd.js changelog?

Reply:
Added a link